quaint/src/connector/queryable.rs

```diff
@@ -99,6 +99,17 @@ pub trait Queryable: Send + Sync {
     fn requires_isolation_first(&self) -> bool;
 }
 
+#[async_trait]
+pub trait Connectable: Queryable {
```

Comment:
Could this trait be moved to js-connectors instead? I don't think quaint's connecteors implement that, `JsConnector` is the only one that needs it.